### PR TITLE
Add pickup info

### PR DESCRIPTION
**Pull request by tomgag**
- Add 5100 info on vas 1073 pickup
- Add SOAP docs to new booking site

### CODE REVIEW

**Comment by bringdev** [comment]
[trello.com/c/cp9yZlP4](https://trello.com/c/cp9yZlP4)

File: data/services_vas.json [hunk 1]
```diff
@@ -2931,8 +2931,8 @@
   "vasFootNotes" : [ ],
   "orderedViaVasCodes" : true
 }, {
-  "vasName" : "Fixed seven-day length of stay",
-  "vasNameNorwegian" : "Fast syv dager liggetid",
+  "vasName" : "Seven days pickup deadline",
+  "vasNameNorwegian" : "Sju dager hentefrist",
```
Comment (larsandremoen) [inline data/services_vas.json:2935]: Should it be `sju` or `Syv`?

Comment (larsandremoen) [reply data/services_vas.json:2935]: Guess both is correct to use, https://ordbokene.no/bm,nn/search?q=sju&scope=ei

**Review by larsandremoen — APPROVED** [review]
LGTM 👍🏽

File: data/services_vas.json (hunk 1 above)
Comment (tomgag) [reply data/services_vas.json:2935]: Not my change, but I guess so.

**Pull request merged**